.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release monorepo packages
+
+on:
+  push:
+    paths:
+      - 'lerna.json'
```

Comment:
When we update lerna.json, in 99.99% of the changes, it's a bump of version to release. The new version will be published.

The other cases won't have any impact, as the publish now doesn't bump the version, but publishes the version in package.json only if it is not already published.